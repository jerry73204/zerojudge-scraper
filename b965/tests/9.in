6 3 4
9 0 2
6 1 2
0 5 1
4 1 8
9 2 6
7 3 5
0 1 0 1