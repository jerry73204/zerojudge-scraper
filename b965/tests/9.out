6 3
9 0 2
6 1 2
0 5 1
4 1 8
9 2 6
7 3 5